index.html

```diff
@@ -0,0 +1,124 @@
+            alt="Trumpet image"
+            width="160"
+            height="70"
+          />
+        </div>
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#something">Features</a></li>
+            <li><a class="nav-link" href="#somethingmore">How It Works</a></li>
+            <li><a class="nav-link" href="#evenmore">Prices</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input name="submit" id="submit" type="submit" value="Submit" />
+        </form>
+      </div>
+
+      <section id="something">
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/1350/1350727.png"
+              alt="Materials"
+              width="60"
+              height="70"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Premium Materials</h2>
+            <p>
+              Our trumpets use the shiniest brass which is sourced locally. This
+              will increase the longevity of your purchase.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/5770/5770775.png"
+              alt="Materials"
+              width="70"
+              height="70"
+            />
+          </div>
+
+          <div class="descrip">
+            <h2>Fast Shipping</h2>
+            <p>
+              We make sure you recieve your trumpet as soon as we have finished
+              making it. We also provide free returns if you are not satisfied.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/3500/3500954.png"
+              alt="Materials"
+              width="60"
+              height="60"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Quality Assurance</h2>
+            <p>
+              For every purchase you make, we will ensure there are no damages
+              or faults and we will check and test the pitch of your instrument.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section id="somethingmore">
+        <div>
+          <iframe
+            width="420"
+            height="315"
+            src="https://www.youtube.com/embed/nODd-8rvJJU"
+          >
+          </iframe>
+        </div>
+      </section>
+      <section id="evenmore">
+        <div>Price1</div>
+        <div>Price2</div>
+        <div>Price3</div>
+      </section>
```

Comment:
Great job using the correct syntax so far 👍  , however, it's best to use descriptives IDs on your code. For example, I think it would be better to have "prices-list" as an id instead of "something" or "evenmore".

Reply:
> Great job using the correct syntax so far 👍 , however, it's best to use descriptives IDs on your code. For example, I think it would be better to have "prices-list" as an id instead of "something" or "evenmore".

It's important to use a descriptive or informative ID tag because it would help your code be easier to read if some other developer has to work with it. It also makes you look more professional. You can find the best CSS and HTML practices here 👍 :  https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md